po/rufus.pot

```diff
@@ -375,11 +375,13 @@ msgctxt "MSG_311"
 msgid "Writing"
 msgstr ""
 
-#. If this FAQ is translated into your language, replace "en-us" with the right language code.
-#. Otherwise, use exactly the same URL.
+#. Language code to use in https://support.endlessm.com/ URLs. For example, the
+#. Spanish FAQ page is at https://support.endlessm.com/hc/es so this should be
+#. "es". If https://support.endlessm.com/ does not have translations for this
+#. language, use "en-us".
```

Comment:
Zendesk seems to redirect (valid) language codes to en-us if the FAQ is not translated, so we could probably simplify this instruction a little. Maybe refer to https://support.zendesk.com/hc/en-us/articles/203761906 as a sanity check.

Reply:
I'll put a reference to that document in, with a note that the brazilian portuguese FAQs are under the 'pt' language code…

Reply:
Oh, and the URL must be lower-case even if the locale ID there is mixed-case.